petastorm/spark/spark_dataset_converter.py

```diff
@@ -139,19 +176,58 @@ def __len__(self):
         """
         return self.dataset_size
 
-    def make_tf_dataset(self):
+    def make_tf_dataset(
+            self,
+            batch_size=32,
```

Comment:
Do not have a very strong opinion here, but I would expect user to explicitly set the batch size - 32 seems too arbitrary.

Reply:
How about changing it to None but keep the internal default to 32. So we can automatically decide the best batch size in the future? If users do not like it, they can always set a value.